src/features/all-drops/components/AllDrops.tsx

```diff
@@ -81,7 +81,7 @@ const COLUMNS: ColumnItem[] = [
 export default function AllDrops() {
   const [searchParams, setSearchParams] = useSearchParams();
 
-  const { setAppModal } = useAppContext();
+  const { setAppModal, refreshTrigger, setRefreshTrigger } = useAppContext();
```

Comment:
Can we add the values types to the context interface please?
![image](https://user-images.githubusercontent.com/19740800/228433921-16cbe486-f74b-45bd-aaa5-ef6ec507f9c2.png)